core/src/main/java/org/lflang/federated/generator/FedTargetConfig.java

```diff
@@ -36,7 +36,8 @@ public FedTargetConfig(LFGeneratorContext context, Resource federateResource) {
     mergeImportedConfig(
         federateResource, context.getFileConfig().resource, context.getErrorReporter());
 
-    clearPropertiesToIgnore();
+    clearPropertiesToIgnore(); // FIXME: add boolean inherit() function to TargetPropertyConfig
```

Comment:
I thought it might be helpful to declare for each target property whether it should be inherited by federates or not. On second though, I think that might be excess generality, so I'll remove the FIXME...